frontend/WikiContrib-Frontend/src/components/activity.js

```diff
@@ -99,7 +99,10 @@ class Activity extends React.Component {
           <React.Fragment>
             {this.state.data.length !== 0 ? (
               <React.Fragment>
-                <h4 style={{marginTop:"1rem"}}>{this.props.username}'s activity on {this.props.date}</h4>
+                <Header className="chart">
+                  {' '}
```

Comment:
Same as my comment below. Not sure why we have `{'  '}` here. We don't need this.